### ISSUE TITLE
Export crop plans

### ISSUE THREAD

**Issue by mstenta** [mstenta/farm_crop_plan#21]
Feedback from Cora at Wolfe's Neck:

> Is there an export function for the crop plan? A way to download the data from one plan might be helpful. Maybe adding a way to filter assets/logs by the specific crop plan would do the trick?

**Comment by ludwa6** [comment]
Yes @Cora: This (as explained in [this issue i posted](https://github.com/mstenta/farm_crop_plan/issues/31) before i saw yours -oops!) is a pressing need for me -and anyone else using the module, i imagine.

As to the doubt you raised in response to my Issue report, @mstenta , i.e.:

> what would it export? The list of planting assets? The list of seeding/transplanting logs? Both? The plan combines information from assets and logs, so it's a bit different than the other CSV exports that exist in farmOS currently

Both logs AND assets related to the Crop Plan for export are relevant and should be included... But if it's too complicated to aggregate those in one export, then perhaps they can (as proposed above) be filtered and exported separately, using existing affordances -yes?

**Comment by mstenta** [comment]
@ludwa6 Yea I think that makes sense. It would be great to devise a common export format that includes everything. Or eve a combination of different export options!

In 2.x there will also be more abilities to access Plan data via the API - so I could see this being implemented outside of farmOS too, potentially - tailored for specific needs. A Python script + [farmOS.py](https://github.com/farmOS/farmOS.py/) could be written to format it into a CSV in a very customized way.

We are currently discussing a core data architecture question, in that regard: https://www.drupal.org/project/farm/issues/3187877
